setup.py

```diff
@@ -19,7 +19,7 @@
             'License :: OSI Approved :: BSD License',
             'Topic :: Software Development :: Libraries :: Python Modules',
             'Topic :: System :: Distributed Computing',
-            'Programming Language :: Python :: 2.7',
+            'Programming Language :: Python :: 2.7 :: Python 3.5',
```

Comment:
adding language should be a new entry see:

https://github.com/pypa/sampleproject/blob/master/setup.py

```
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3.5',
```